src/test/storage/storage_manager_test.cpp

```diff
@@ -115,4 +116,35 @@ TEST_F(StorageManagerTest, ListViewNames) {
   EXPECT_EQ(view_names[1], "second_view");
 }
 
+TEST_F(StorageManagerTest, Print) {
+  std::ostringstream output;
+  auto& sm = StorageManager::get();
+  sm.print(output);
+
+  auto output_string = output.str();
+
+  EXPECT_TRUE(output_string.find("===== Tables =====") != std::string::npos);
+  EXPECT_TRUE(output_string.find("==== table >> first_table << (0 columns, 0 rows in 0 chunks)") != std::string::npos);
```

Comment:
Mmmh, testing with an empty table doesn't do anything except for making the coverage misleadingly green. At least put in a simple table. Also, why are you `find`ing in the string instead of comparing it 1:1?